Builders/Windows/SetupProject/Version.cs

```diff
@@ -1,4 +1,4 @@
 using System.Reflection;
 
-[assembly: AssemblyVersion("5.0.7.1")]
-[assembly: AssemblyFileVersion("5.0.7.1")]
\ No newline at end of file
+[assembly: AssemblyVersion("5.0.8.1")]
```

Comment:
don't change it manually next time, I have a script that changes all the needed file based on this one